mteb/evaluation/evaluators/RerankingEvaluator.py

```diff
@@ -147,22 +136,17 @@ def compute_metrics_individual(self, model):
         """
         # using encode_queries and encode_corpus functions if they exists,
         # which can be defined by users to add different instructions for query and passage conveniently
-        encode_queries_func = (
-            model.encode_queries if hasattr(model, "encode_queries") else model.encode
-        )
-        encode_corpus_func = (
-            model.encode_corpus if hasattr(model, "encode_corpus") else model.encode
-        )
+
         if self.evaluator_type == "standard":
             results = self._encode_candidates(
-                encode_queries_func=encode_queries_func,
-                encode_corpus_func=encode_corpus_func,
+                encode_queries_func=model.encode,
+                encode_corpus_func=model.encode,
```

Comment:
This seems like a mistake? Either is should just be 1 encode or we should make it two different ones

Reply:
I kept the current setup to avoid a refactor, but will do this now. The main issue is that this version uses prompts, whereas the version on the main branch does not. Should I remove the prompts to align the results with the main branch?

Reply:
Let us remove the prompt to avoid a major bump but create an issue on it and mark it v2.0.0

Reply:
Would be great with to invalidate "results on [these tasks] before version X.X.X, should be invalid" going forward.

For now, let us include the prompts (that is intended behavior, after all).

Reply:
> Would be great with to invalidate "results on [these tasks] between version X.X.X and X.X.X, should be invalid" going forward.

You mean I should add warning when loading results in splits for reranking tasks?

> For now, let us include the prompts (that is intended behavior, after all).

You mean #1300? Because current implementation passing instructions to wrappers correctly, like previously

Reply:
> You mean I should add warning when loading results in splits for reranking tasks?

More of a general concern. Sometimes, we find a mistake for specific versions of MTEB and we would like to not include those in the benchmark.